src/drivers/gnss/septentrio/septentrio.cpp

```diff
@@ -140,7 +140,7 @@ constexpr const char *k_frequency_25_0hz = "msec40";
 constexpr const char *k_frequency_50_0hz = "msec20";
 
 px4::atomic<SeptentrioDriver *> SeptentrioDriver::_secondary_instance {nullptr};
-uint32_t SeptentrioDriver::k_supported_baud_rates[] {0, 38400, 57600, 115200, 230400, 460800, 500000, 576000, 921600, 1000000, 1500000};
+uint32_t SeptentrioDriver::k_supported_baud_rates[] {0, 38400, 57600, 115200, 230400, 460800, 921600};
```

Comment:
Intentional?

Reply:
Pending https://github.com/PX4/PX4-Autopilot/pull/23415.

Reply:
Dropped in the latest rebase.